CONTRIBUTING.md

```diff
@@ -87,38 +87,9 @@ The test module, clusterctl, and experiments do not provide any backward compati
-merged into the main branch.
-
-Any backport MUST not be breaking for either API or behavioral changes. 
-
-We generally allow backports of following changes to all supported branches:
-- Critical bugs fixes, security issue fixes, or fixes for bugs without easy workarounds.
-- Dependency bumps for CVE (usually limited to CVE resolution; backports of non-CVE related version bumps are considered exceptions to be evaluated case by case)
-- Cert-manager version bumps (to avoid having releases with cert-manager versions that are out of support, when possible)
-- Changes required to support new Kubernetes versions, when possible. See [supported Kubernetes versions](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions) for more details.
-- Changes to use the latest Go patch version to build controller images.
-- Changes to bump the Go minor version used to build controller images, if the Go minor version of a supported branch goes out of support (e.g. to pick up bug and CVE fixes). 
-  This has no impact on folks importing Cluster API as we won't modify the version in `go.mod` and the version in the `Makefile` does not affect them.
-
-We generally allow backports of following changes only to the latest supported branch:
-- Improvements to existing docs (the latest supported branch hosts the current version of the book)
-- Improvements to CI signal
-- Improvements to the test framework
-
-While we recommend to target following type of changes to the next minor release, CAPI maintainers will also consider
-exceptions for backport of following changes only to the latest supported branch:
-- Enhancements or additions to experimental Cluster API features, with the goal of allowing faster adoption and iteration;
-  Please note that stability of the branch will always be top priority while evaluating those PRs, and thus approval
-  requires /lgtm from at least two maintainers that, on top of checking that the backport is not introducing any breaking
-  change for either API or behavior, will evaluate if the impact of those backport is limited and well-scoped e.g. 
-  by checking that those changes should not touch non-experimental code paths like utils and/or by applying other 
-  considerations depending on the specific PR.
-
-Like any other activity in the project, backporting a fix/change is a community-driven effort and requires that someone volunteers to own the task. 
-In most cases, the cherry-pick bot can (and should) be used to automate opening a cherry-pick PR.
-
-We generally do not accept backports to Cluster API release branches that are [out of support](https://github.com/kubernetes-sigs/cluster-api/blob/main/CONTRIBUTING.md#support-and-guarantees).
+Pull Requests against the main branch can be backported using `/cherry-pick` prow command. Any backport MUST NOT be breaking for API or behavioral changes.
+
```

Comment:
```suggestion

We usually backport all critical bug fixes, security fixes (including dependency bumps), Golang minor and patch bumps (to pick up fixes), cert-manager bumps (to avoid using out-of-support versions), changes to support new Kubernetes minor versions when possible ([supported Kubernetes versions](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions)), improvements to documentation (for the book), the CI signal and the test framework.

```

What do you think about adding this sentence, just to illustrate what we usually backport and for which reasons. This restricts in no way what we *can* backport.
I think it's also still somewhat concise and not too verbose.

Reply:
^^ @fabriziopandini @vincepri WDYT?

Reply:
nits
```suggestion

We usually backport critical bug fixes, security fixes (including dependency bumps and golang minor and patch bumps), cert-manager bumps (to avoid using out-of-support versions), changes to support new Kubernetes minor versions when possible ([supported Kubernetes versions](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions)), and also improvements to documentation, the CI signal and the test framework.

Everything else is considered case by case.
```